tests/testthat/test-bayesianpredictionperformance.R

```diff
@@ -0,0 +1,425 @@
+  test_that("Prior and posterior plot matches", {
+    plotName <- results[["results"]][["priorAndPosteriorPlot"]][["data"]]
+    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+    jaspTools::expect_equal_plots(testPlot, "prior-and-posterior-plot-1")
+  })
+
+  test_that("Running means plot matches", {
+    plotName <- results[["results"]][["rmPlot"]][["data"]]
+    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+    jaspTools::expect_equal_plots(testPlot, "running-means-plot-1")
+  })
+
+  test_that("Observed-Expected Ratio Meta-Analysis Summary table results match", {
+    table <- results[["results"]][["summaryTable"]][["data"]]
+    jaspTools::expect_equal_tables(table,
+                                   list(1.09088086726799, 0.88575857156341, 0.285932954156389, 1.34586691302466,
+                                        2.45439237587575))
+  })
+}
+
+# O:E: Poisson/log (+ asymmetry tests)
+{
+  options <- analysisOptions("BayesianPredictionPerformance")
+  options$chains <- 2
+  options$diagnosticsAcPlot <- TRUE
+  options$diagnosticsGelmanRubinPlot <- TRUE
+  options$diagnosticsRmPlot <- TRUE
+  options$exportColumns <- FALSE
+  options$exportCstat <- "JaspColumn_17_Encoded"
+  options$exportCstatlCI <- "JaspColumn_18_Encoded"
+  options$exportCstatuCI <- "JaspColumn_19_Encoded"
+  options$exportOE <- "JaspColumn_20_Encoded"
+  options$exportOElCI <- "JaspColumn_21_Encoded"
+  options$exportOEuCI <- "JaspColumn_22_Encoded"
+  options$forestPlot <- TRUE
+  options$forestPlotEstimates <- TRUE
+  options$forestPlotLabels <- TRUE
+  options$funnelAsymmetryTest <- TRUE
+  options$funnelAsymmetryTestDebrayFIV <- TRUE
+  options$funnelAsymmetryTestEggerFIV <- TRUE
+  options$funnelAsymmetryTestEggerUW <- TRUE
+  options$funnelAsymmetryTestMacaskillFIV <- TRUE
+  options$funnelAsymmetryTestMacaskillFPV <- TRUE
+  options$funnelAsymmetryTestPeters <- TRUE
+  options$funnelAsymmetryTestPlot <- TRUE
+  options$inputCI <- list()
+  options$inputE <- "e.events"
+  options$inputLabels <- ""
+  options$inputMeasure <- ""
+  options$inputN <- ""
+  options$inputO <- "n.events"
+  options$inputSE <- ""
+  options$linkCstat <- "normal/logit"
+  options$linkOE <- "poisson/log"
+  options$measure <- "OE"
+  options$method <- "Restricted ML"
+  options$priorAndPosteriorPlot <- TRUE
+  options$priorMuNMeam <- 0
+  options$priorMuNSD <- 10
+  options$priorTau <- "priorTauU"
+  options$priorTauTDf <- 3
+  options$priorTauTLocation <- 0
+  options$priorTauTMax <- 10
+  options$priorTauTMin <- 0
+  options$priorTauTScale <- 1.5
+  options$priorTauUMax <- 2
+  options$priorTauUMin <- 0
+  options$sample <- 5000
+  set.seed(1)
+  dataset <- structure(list(Study = structure(c(14L, 2L, 8L, 7L, 9L, 5L, 12L,
+                                                11L, 10L, 20L, 3L, 18L, 19L, 21L, 13L, 20L, 15L, 1L, 1L, 17L,
+                                                6L, 4L, 16L), .Label = c("Barilli", "Biancari", "Borde", "Borracci",
```

Comment:
Add a CSV file to tests and load it? Same for all the others.

Reply:
how about loading it with ```data("EuroSCORE", package = "metamisc")``` ?